ui/components/institutional/signature-mismatch-banner/signature-mismatch-banner.tsx

```diff
@@ -55,26 +46,12 @@ const MMISignatureMismatchBanner: React.FC = memo(() => {
     return null;
   }
 
-  return (
-    <Box
-      display={Display.Flex}
-      width={BlockSize.Full}
-      padding={4}
-      backgroundColor={BackgroundColor.primaryMuted}
-    >
-      <Icon
-        name={IconName.Info}
-        color={IconColor.infoDefault}
-        marginRight={2}
-      />
-      <Text variant={TextVariant.bodyXs} color={TextColor.textDefault}>
-        {t('mismatchAccount', [
-          shortenAddress(selectedAccount?.address),
-          shortenAddress(fromAccount?.address),
-        ])}
-      </Text>
-    </Box>
-  );
+  const message = t('mismatchAccount', [
+    shortenAddress(selectedAccount?.address),
+    shortenAddress(fromAccount?.address),
+  ]);
+
+  return <BannerAlert title={message} />;
```

Comment:
```suggestion
  return <BannerAlert marginTop={3} title={message} />;
```

Reply:
```suggestion
  return <BannerAlert marginTop={3} title={message} />;
```
thanks for the update in `ui/pages/confirmations/confirm/confirm.tsx`! re-adding this to support the margin